docs/src/app/components/Divider/README.md

```diff
@@ -0,0 +1,5 @@
+## Divider
+
+<a href="https://www.google.com/design/spec/components/dividers.html" target="_blank">Dividers</a> group and separate content within lists and page layouts. The divider is a thin rule, lightweight yet sufficient to distinguish content visually and spatially.
```

Comment:
Mm, no luck with that. Maybe it's alright if it just opens in the same window?

It kind of threw me off the first time I tried it but maybe it's fine.